.gitignore

```diff
@@ -16,3 +16,8 @@ config.json
 coverage.*
 lib-cov
 complexity.md
+
+#Ignore vim temporary and swap files
```

Comment:
Add these to your global gitignore that is specific to your environment, not to the project specific one. See more information here:

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore